src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -363,6 +376,15 @@ static void _applyProperty(SvgNode* node, Shape* vg, float vx, float vy, float v
         vg->stroke(style->stroke.paint.color.r, style->stroke.paint.color.g, style->stroke.paint.color.b, style->stroke.opacity);
     }
 
+    if (node->style->clipPath.node && node->style->clipPath.node->objectBBox) {
+        vg->bounds(&vx, &vy, &vw, &vh, false);
+        if (auto strokeW = vg->strokeWidth()) {
+            vx += 0.5f * strokeW;
+            vy += 0.5f * strokeW;
+            vw -= strokeW;
+            vh -= strokeW;
+        }
```

Comment:
This code started to duplicate. It seems to be possible to extract by method.

Reply:
as I marked in some places, I'd need some other version of bounds api, so this part is only a temporary solution

Reply:
@JSUYA I'll take care of that in a separate pr, since I need to solve the problem with bounds without any strokes for the scene

Reply:
Doesn't this simple work?
```
auto stroke = vg->strokeWidth(); 
vg->stroke(0);  //excluding stroke width.
vg->bounds(&vx, &vy, &vw, &vh, false);
vg->stroke(stroke);
```